x-pack/plugins/infra/public/store/remote/log_summary/operations/load.ts

```diff
@@ -22,7 +22,7 @@ export const loadSummaryActionCreators = createGraphqlOperationActionCreators<
 
```

Comment:
See previous comments, I believe this results in a prefix `x-pack/infra/remote/summary` on action names, can we use `log_summary`?

![image](https://user-images.githubusercontent.com/189073/44408616-03440400-a561-11e8-93ef-1b54a020a61d.png)